src/status_im/ui/screens/chat/styles/message/message.cljs

```diff
@@ -136,23 +140,33 @@
 
 (styles/defn emoji-message
   [{:keys [incoming-group]}]
-  {:font-size 40
-   :desktop   {:line-height 46}
-   :margin-top (if incoming-group 4 0)})
+  {:font-size    40
```

Comment:
is that emoji only font-size? 40 is hardcore, can we lower it to 28

Reply:
Yes, it is only emoji size - I can do it.